docs/gaia/deploy-testnet.md

```diff
@@ -44,7 +44,7 @@ gaiacli keys add validator
 # Add that key into the genesis.app_state.accounts array in the genesis file
 # NOTE: this command lets you set the number of coins. Make sure this account has some coins
 # with the genesis.app_state.staking.params.bond_denom denom, the default is staking
-gaiad add-genesis-account $(gaiacli keys show validator -a) 1000000000stake,1000000000validatortoken
+gaiad add-genesis-account $(gaiacli keys show validator -a) 1000000000atom,1000000000validatortoken
```

Comment:
testnets _do not_ use atoms but actually use an imaginary token named `stake` (aka this PR change should be reversed)